pillar.example

```diff
@@ -5,6 +5,7 @@ nfs:
   mount_opts: noauto,ro
   persist_unmount: true
   persist_mount: false
+  disabled: false
```

Comment:
I think I prefer to have `enabled: false` like in many other formulas.

Reply:
I suppose `enabled: true` then, so NFS remains enabled by default unless disabled in configuration?

Reply:
Yes, the key `enabled` is better. It avoid the reverse logic `the service is enabled when disabled is false`.